packages/wp-dashboard/src/api/utils/reshapeStoryObject.js

```diff
@@ -32,16 +32,16 @@ export default function reshapeStoryObject(originalStoryData) {
     preview_link: previewLink,
     edit_link: editStoryLink,
     story_poster: storyPoster,
-    _embedded: {
-      author = [{ name: '', id: 0 }],
-      'wp:lock': lock = [{ locked: false }],
-      'wp:lockuser': lockUser = [{ id: 0, name: '' }],
-    } = {},
+    _embedded: { author, 'wp:lock': lock } = {},
     _links: links = {},
   } = originalStoryData;
   if (!id) {
     return null;
   }
+
+  const { locked = false, user: lockUser = { id: 0, name: '' } } =
+    lock?.[0] || {};
+  const storyAuthor = author?.[0] || { id: 0, name: '' };
```

Comment:
Note that js unit tests now need updating

Reply:
Fixed.